src/main/java/com/netgrif/application/engine/workflow/service/TaskService.java

```diff
@@ -698,6 +698,13 @@ public Task save(Task task) {
         return task;
     }
 
+    @Override
+    public List<Task> save(List<Task>  tasks) {
+        tasks = taskRepository.saveAll(tasks);
+        tasks.forEach(task -> elasticTaskService.index(this.taskMappingService.transform(task)));
```

Comment:
does a bulk elastic index method exist? I have a feeling something like that was implemented at some point.